csharp/lib/AsyncClient.cs

```diff
@@ -94,6 +98,16 @@ private void FailureCallback(ulong index)
 
     #region FFI function declarations
 
+    public enum RequestType : uint
```

Comment:
can we remove that and share the RequestType in the header file? 
I'm not sure we need to convert from RequestType to RequestType to Cmd when it's 1:1:1.

Reply:
It is possible. We can define the enum into a C header file and then [import it to C#](https://stackoverflow.com/questions/3457646/using-enum-in-c-and-c-sharp-via-c-header) project and to rust (using [bindgen](https://github.com/rust-lang/rust-bindgen?tab=readme-ov-file)).
We can also you protoc to do this for us and suppress a warning about not-FFI-safe enum.